docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst

```diff
@@ -55,15 +57,77 @@ on our `feedback form <https://docs.google.com/forms/d/e/1FAIpQLSeCED9TFQsH-1u4Q
 
    * - This is...
      - This isn't...
-   * -
-     -
+   * - A ‘living’ document which will be regularly reviewed and updated to reflect evolving language and research practices. 
+     - An immutable document that should be treated as all-knowing and final.
+   * - An accessible guide intended for everyone’s use, regardless of your familiarity with inclusive language practices. 
+     - An intimidating guide that is full of jargon and can only be used by DEI experts. 
+   * - A safe space for people to make mistakes, ask questions, and learn new things! 
+     - A space where people need to police what they are saying or how they are speaking for fear of judgment.
+   * - Using our voice on the Simulation Science team to platform the terms we find most accurate and inclusive. 
+     - Putting words into other people’s mouths (such as racial or ethnic categorizations in external data sources). 
+
 
 Guiding principles
 ------------------
 
-.. todo::
+Our guiding principles for this document can be summarized in three points: 
+
+**1. This work is important**
+
+Inclusive language use is important, and we want to begin by acknowledging the potential harm that using 
```

Comment:
Just brainstorming some ideas of things we could say besides "inclusive language" that would be a bit broader:
- Inclusive research practices
- Equity-focused research practices
- Equity-focused data analyses
- Health disparities research?